fake/src/qtfirebaseauthenticate.h

```diff
@@ -0,0 +1,68 @@
+#ifndef QTFIREBASE_AUTHENTICATE_H
+#define QTFIREBASE_AUTHENTICATE_H
+
+#ifdef QTFIREBASE_BUILD_AUTHENTICATE
+
+#include "src/qtfirebase.h"
+
+#if defined(qFirebaseAuthenticate)
+#undef qFirebaseAuthenticate
+#endif
+#define qFirebaseAuthenticate (static_cast<QtFirebaseAuthenticate*>(QtFirebaseAuthenticate::instance()))
+
+#include <QObject>
+#include <QVariantMap>
+#include <QVariantList>
+
+class QtFirebaseAuthenticate : public QObject
```

Comment:
I can't see QtFirebaseAuthenticate being registered as a QML type anywhere - shouldn't it have it's own QML type?